portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,34 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Welcome to Alec's portfolio!</h1>
+      <p>This is my portfolio! I'm currently a sophomore at NYU and doing a joint major in Computer Science and Economics.</p>
+      <p> I was born in Switzerland and am also Colombian and American. I'm a big fan of sports (soccer, NFL, NBA, MLB, NHL) and am also growing my interest in cooking and baking.</p>
+      <p> I love memes so don't be suprised to see a lot of memes here.</p>
+      <p> Here is a list of my favorite projects that I've done: 
+          Virtual Moped Project,
+          Binary Search Tree Balancer,
+          Tic Counter. 
+
+           I plan on making an app game about Le Monke.</p>
+      <p> Link to my github: https://github.com/elongateddmuskrat</p>
```

Comment:
Sorry, I should have mentioned this earlier. You will want your links in your HTML wrapped in the `<a href=...>...</a>` tag like this:
```
<a href="https://github.com/elongateddmuskrat">https://github.com/elongateddmuskrat</a>
```
Same for your LinkedIn link in the next line.